library/wizard/terminate_reason.py

```diff
@@ -0,0 +1,52 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class TerminateReasonLibrary(models.TransientModel):
+    _inherit = "terminate.reason"
+
+    info = fields.Text("Message", help="Enter message")
+
+    @api.model
+    def default_get(self, fields):
+        """Override method to display message if student has issued book
+        while terminate student"""
+        res = super(TerminateReasonLibrary, self).default_get(fields)
+        student_obj = self.env["student.student"].browse(
+            self._context.get("active_id")
+        )
+        library_card_rec = self.env["library.card"].search(
+            [("student_id", "=", student_obj.id), ("state", "=", "running")]
+        )
+        book_issue_rec = self.env["library.book.issue"].search(
+            [
+                ("state", "in", ["issue", "reissue"]),
+                ("student_id", "=", student_obj.id),
+            ]
+        )
+        card_info = ""
+        if library_card_rec:
+            card_info += (
+                "The student has library card assigned with number"
+                + " "
+                + library_card_rec.code
+                or ""
+            )
+        if book_issue_rec:
```

Comment:
@ForamSerpentCS 
Remove if condition. As it will not go in for loop if **book_issue_rec** is not there.
don't need **if** here.